webui/src/lib/components/repository/compareBranchesActionBar.jsx

```diff
@@ -3,59 +3,25 @@ import {refs as refsAPI} from "../../../lib/api";
 import {RefTypeBranch} from "../../../constants";
 import {ActionGroup, ActionsBar, AlertError, RefreshButton} from "../controls";
 import {MetadataFields} from "./changes";
-import {useRouter} from "../../hooks/router";
-import RefDropdown from "./refDropdown";
-import {ArrowLeftIcon, ArrowSwitchIcon, GitMergeIcon} from "@primer/octicons-react";
-import OverlayTrigger from "react-bootstrap/OverlayTrigger";
-import Tooltip from "react-bootstrap/Tooltip";
+import {GitMergeIcon} from "@primer/octicons-react";
 import Button from "react-bootstrap/Button";
 import Modal from "react-bootstrap/Modal";
 import Form from "react-bootstrap/Form";
 import {FormControl, FormHelperText, InputLabel, MenuItem, Select} from "@mui/material";
+import CompareBranchesSelection from "./compareBranchesSelection";
 
 const CompareBranchesActionsBar = (
-    {repo, reference, onSelectRef, compareReference, onSelectCompare, doRefresh, isEmptyDiff}
+    {repo, reference, compareReference, baseSelectURL, doRefresh, isEmptyDiff}
 ) => {
-    const router = useRouter();
```

Comment:
The `Compare` and `Create PR` pages are both using the same component of branches selection,
Only with different baseURL and slightly different Ref params.
So I've unified them.